algorithms/ar-isrt/cpp/insertion_sort.cpp

```diff
@@ -0,0 +1,27 @@
+#include<iostream>
+using namespace std;
+
+int* insertionSort(int arr[], int n){
+	int position,valueToInsert;
```

Comment:
Codacy found an issue: [The scope of the variable 'position' can be reduced.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6424275)